mod/thewire/views/default/thewire.js

```diff
@@ -27,15 +27,16 @@ elgg.thewire.textCounter = function(textarea, status, limit) {
 
 	var remaining_chars = limit - $(textarea).val().length;
 	status.html(remaining_chars);
+	var $submit = $(textarea).closest('.elgg-form-thewire-add').find('#thewire-submit-button').eq(0);
```

Comment:
i know the use cases you are referring but you could also have multiple id's, however less likely than multiple spans. Luckily both are in control of the thewire plugin in this case, so we should be save... i will remove them from both locations